definitions/ext-ibmmqmanager/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: EXT
+type: IBMMQQUEUEMANAGER
```

Comment:
The folder name needs to match the domain type: 

> EXT IBMMQQUEUEMANAGER... expected ext-ibmmqqueuemanager

Reply:
@IreneMLC - fixed!